CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ Releases
 This page documents the releases of Sleeper. Performance figures for each release
 are available [here](docs/13-system-tests.md#performance-benchmarks)
 
+## Version 0.23.0
+
+This contains the following improvements:
+
+Tables:
+
+- Updated queues used by scheduled operations to be FIFO queues. This is to ensure only one scheduled action is running at a time for each Sleeper table.
+
+State store:
+
+- Added a new state store type - `DynamoDBTransactionLogStateStore`.
+  - This state store intends to improve on update times by only dealing with changes that have happened since the last state. This is an improvement over the `S3StateStore`, which always loads the entire state of the state store, regardless of how little has changed since the last revision.
+
+Compactions:
+
+- Added a new lambda to batch-apply finished compaction jobs.
```

Comment:
The new lambda doesn't apply the jobs in batches, it lets the results of a compaction job be committed asynchronously.